src/Client.ts

```diff
@@ -39,6 +39,22 @@ export default class LavasfyClient {
         this.nodes.set(options.id, new Node(this, options));
     }
 
+    public removeNode(query: string): boolean {
+        if (!this.nodes.size) throw new TypeError("No nodes available, please add a node first...");
+        if (!query) throw new TypeError("Provide a valid node identifier to delete it");
+
+        if (query && this.nodes.has(query)) return this.nodes.delete(query);
```

Comment:
you can remove the if statement here

Reply:
because
![image](https://user-images.githubusercontent.com/52526068/118400427-d11d4d80-b68b-11eb-8785-3df173022645.png)
[ref](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/delete#return_value)